apps/ejabberd/src/mod_amp.erl

```diff
@@ -112,6 +113,8 @@ amp_check_packet(_, _, Acc, _, _) ->
     Acc.
 
 strip_amp_el_from_request(Packet) ->
+    % this will probably be removed - we have accumulator so we won't need anymore to store amp markers
```

Comment:
According to **Elvis**:
> Line 116 is too long:     % this will probably be removed - we have accumulator so we won't need anymore to store amp markers.